.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.15.x, 1.16.x]
+        go-version: [1.16.x, 1.17.x]
```

Comment:
Dropped 1.15 since Lotus in `v1.11.2` isn't compatible with it (needs >1.15).
Added the latest version too for tests.